tests/Unit/API/Site/Controllers/MerchantCenter/PriceBenchmarksControllerTest.php

```diff
@@ -114,4 +123,38 @@ public function test_get_price_benchmarks() {
 		// The expected shape should pass once the implementation is updated.
 		$this->assertSameSets( $expected, $response->get_data(), 'The response data should match the expected structure.' );
 	}
+
+	public function test_get_price_benchmarks_summary() {
+		// Mock the benchmark data.
+		$mock_benchmark_data = [
+			'total_products' => 0,
+			'price_unknown'  => 0,
+			'price_lower'    => 0,
+			'price_similar'  => 0,
+			'price_higher'   => 0,
+		];
+
+		// Configure the mocked methods.
+		$this->price_benchmarks->expects( $this->once() )
+			->method( 'get_summary' )
+			->willReturn( $mock_benchmark_data );
+
+		// Simulate a GET request.
+		$response = $this->do_request( self::ROUTE_PRICE_BENCHMARKS_SUMMARY, 'GET' );
+
+		// Expected shape once data from the two queries are stitched together.
+		$expected = [
+			'total_products' => 0,
+			'price_unknown'  => 0,
+			'price_lower'    => 0,
+			'price_similar'  => 0,
+			'price_higher'   => 0,
+		];
```

Comment:
This inline comment is copy 🍝 from a different test, I think. And the `$expected` array could be removed since it's identical to the `$mock_benchmark_data`, and we could use that in both places.